requirements/bench.txt

```diff
@@ -35,3 +35,5 @@ silence_tensorflow  # for olymp
 tensorflow_probability  # for olymp
 bayes-optim==0.2.5.5
 nlopt
+pybullet==3.2.2
+box2d-py==2.3.5
```

Comment:
can we use >= ? I'm afraid of incompatibility issues, so >= gives some flexibility... but may remove some robustness, so depends